fsck/fsck.c

```diff
@@ -1478,6 +1478,12 @@ static char *bytes_to_human_readable(size_t bytes)
 	}
 
 	quoti = (unsigned int)(bytes / (1ULL << shift));
+
+	if(i > =  sizeof(units)/sizeof(units[0])) {
```

Comment:
How about moving these statements to the line 1479? The variable quoti could be wrong for the size which is greater than 1024PB.

Reply:
OK, this is my carelessness. Thanks